elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -609,6 +610,22 @@ elytron.simple-role-decoder.remove=Remove operation for the RoleDecoder
 # Attributes
 elytron.simple-role-decoder.attribute=The name of the attribute from the identity to map directly to roles.
 
+elytron.source-address-role-decoder=Definition of a RoleDecoder that maps roles based on the IP address of a remote client.
+# Operations
+elytron.source-address-role-decoder.add=Add operation for the RoleDecoder
+elytron.source-address-role-decoder.remove=Remove operation for the RoleDecoder
+# Attributes
+elytron.source-address-role-decoder.source-address=The IP address to match. Exactly one of 'address' and 'pattern' must be specified.
+elytron.source-address-role-decoder.pattern=A regular expression that specifies the IP address to match. Exactly one of 'address' and 'pattern' must be specified.
+elytron.source-address-role-decoder.roles=The list of roles to assign if the IP address of the remote client matches.
+
+elytron.aggregate-role-decoder=Definition of a RoleDecoder that is an aggregation of other role decoders.
```

Comment:
you mention both `RoleDecoder` and `role decoder` while the the name of the resources are `source-address-role-decoder` and `aggregate-role-decoder`. The name of the resources and their descriptions should be consistent

Reply:
Same as above.